.babelrc

```diff
@@ -0,0 +1,22 @@
+{
+    "presets": [
+        ["env", {
+            "entry": ["babel-polyfill", "./src/client.js"],
+            "useBuiltIns": true,
+            "targets": {
+                "browsers": [
+                    "chrome >= 53",
+                    "firefox >= 45.0",
+                    "ie >= 11",
+                    "edge >= 37",
+                    "safari >= 9",
+                    "opera >= 40",
+                    "op_mini >= 18",
+                    "Android >= 7",
+                    "and_chr >= 53",
+                    "and_ff >= 49",
+                    "ios_saf >= 10"]
+            }
+        }], ["razzle"]
```

Comment:
This isn't how we add the razzle babel config, need to do something [like this](https://github.com/jaredpalmer/razzle/#customizing-babel-config).